core/services/fluxmonitorv2/flux_monitor.go

```diff
@@ -174,11 +174,15 @@ func NewFromJobSpec(
 		return nil, err
 	}
 
+	gasLimit := cfg.EvmGasLimitDefault()
```

Comment:
I actually think it would be ideal to have per-job-type in addition to per-job gas limits. So the override precedence would be:

```
job-specific > job-type-specific > global default
```

This becomes cleaner with the new config since each job type can have a `GasLimit` field if appropriate. For now we probably just want env vars like `FM_GAS_LIMIT`, `KEEPER_GAS_LIMIT` etc

Reply:
As discussed offline, this will be done in a next PR as a separate feature.